src/Stratis.Bitcoin/Configuration/Settings/ConnectionManagerSettings.cs

```diff
@@ -122,5 +219,14 @@ public void Load(NodeSettings nodeSettings)
 
         /// <summary>Connections number after which burst connectivity mode (connection attempts with no delay in between) will be disabled.</summary>
         public int BurstModeTargetConnections { get; internal set; }
+
+        /// <summary><c>true</c> to sync time with other peers and calculate adjusted time, <c>false</c> to use our system clock only.</summary>
+        public bool SyncTimeEnabled { get; private set; }
+
+        /// <summary>The node's user agent.</summary>
+        public string Agent { get; private set; }
```

Comment:
This is now duplicated with the one in NodeSettings.

Reply:
This is not duplication. The Agent on the NodeSettings is per definition different from the one on ConnectionSettings as the latter includes a prefix. It is bad programming practice to use the same variable for two things that are defined differently (i.e. includes a prefix vs no prefix).

Reply:
sounds good